### DISCUSSION TITLE
Include .zip build in GitHub release artifacts

### DISCUSSION THREAD

**Discussion by lppedd** [JaneWardSandy/eclipse-pde-partial-idea#18] [General]
Unfortunately the Plugin Marketplace is slow to approve plugins at times.  
I'd say it would be cool to share the plugin's .zip of each release on GitHub.

**Comment by JaneWardSandy** [comment]
Under normal circumstances, the review will be passed around 24 hours, and submissions during Jetbrains working hours can occasionally pass within 1 hour.
I try to get stuck in time to submit. If it is still too slow, I will consider putting it in the Release attachment
